packages/build-utils/src/errors.ts

```diff
@@ -61,14 +61,14 @@ export function getPrettyError(obj: {
     }
 
     return new NowBuildError({
-      code: 'DEV_VALIDATE_CONFIG',
+      code: 'invalid_vercel_config',
```

Comment:
Aw, I liked `INVALID_LOCAL_CONFIG` better.

Reply:
<img src="https://media.giphy.com/media/KOB9yLMfyn9ygrwFn7/giphy.gif" />

Reply:
Sadly, this code is already used for a different error here: https://github.com/vercel/vercel/blob/cfae7ec3c291d1802fb6eec6642a0ebf792fb71d/packages/cli/src/util/errors.ts#L38

Reply:
Yeah, I accidentally copy-n-pasted `INVALID_LOCAL_CONFIG` when I meant `INVALID_VERCEL_CONFIG` as you had it. I just like my `code` in `CAPS`.

Reply:
I guess Node.js error codes are uppercase so lets keep it uppercase [e4a89b7](https://github.com/vercel/vercel/pull/8623/commits/e4a89b72d967a08d0e28799c6c6b067067018d7a)